src/proc_image_processing/cpu/filters/white_noise_takedown.h

```diff
@@ -26,10 +26,10 @@ namespace proc_image_processing {
       high_r_("HighR", 0, 0, 255, &parameters_),
       view_channel_("Channel_view", 0, 0, 3, &parameters_,
         "0=ALL, 1=Blue, 2=Green, 3=Red") {
-      SetName("WhiteNoiseTakedown");
+      SetName("WhiteNoiseTakeDown");
```

Comment:
I think this should stay unmodified. "takedown" is a singleword, hence only the T of takedown should be in capital letter. In other words, it's "white noise takedown" and not "white noise take down". Camelcasing should reflect the spaces in between words.

Reply:
Take Down suits better in this case since we are taking the white noise down. We could also name this WhiteNoiseRemoval instead.